src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -14,10 +14,14 @@
         <!-- tabs -->
         <ul class="nav navbar-nav">
           <li ng-class="{active: sidebar.section == 'data'}" ng-show="sidebar.showData">
-            <a class="navbar-link active" ng-click="sidebar.section='data'">Data</a>
+            <a class="vis-editor-subnav-link"
+               ng-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
+               ng-click="sidebar.section='data'">Data</a>
```

Comment:
Hey Peter, the [most recent version](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md) of the style guide exemplifies this in the examples. Should I update the descriptions to point this out more, or do you think this is enough?

This is a pretty recent change to the style guide. You can check out [this PR](https://github.com/elastic/kibana/pull/7521) for some context.

I'm using Sublime, so I'm not sure why WebStorm would be unhappy with that formatting... is this something you can look into fixing on your end? Let me know if you want to Zoom about this and tackle it together.